assets/css/stars.css

```diff
@@ -0,0 +1,79 @@
+.rating {
```

Comment:
Great idea to move this into a separate file, to allow enqueuing it in a few places.